components/authn-service/constants/constants.go

```diff
@@ -11,3 +11,13 @@ const AdvisoryLockID = 8701 // Usher Baby!
 
 // LocalConnectorID is the ID used for dex' local user connector
 const LocalConnectorID = "local"
+
+// UnassignedProjectID represents the project ID for
+// resources (tokens) without projects
+const UnassignedProjectID = "(unassigned)"
+
+// AllProjectsExternalID is the representation of all projects to downstream services
+const AllProjectsExternalID = "*"
+
+// PgURL is the URL to connect to the authn DB
+const PgURL = "postgresql://postgres@127.0.0.1:5432/authn_test?sslmode=disable"
```

Comment:
I see the value in extracting this, but also it feels sort of funny to have a test value lumped in here 🤔

Reply:
Maybe: `TestPgURL` will clarify